magellan-flow-sample/src/main/java/com/wealthfront/magellan/sample/Expedition.kt

```diff
@@ -0,0 +1,16 @@
+package com.wealthfront.magellan.sample
+
+import android.content.Context
+import com.wealthfront.magellan.core.Journey
+import com.wealthfront.magellan.sample.databinding.ExpeditionBinding
+
+class Expedition : Journey<ExpeditionBinding>(ExpeditionBinding::inflate, ExpeditionBinding::container) {
```

Comment:
lol we're actually calling this `Expedition` 👍

Reply:
yup 😛